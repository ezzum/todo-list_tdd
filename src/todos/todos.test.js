import { mount } from 'enzyme';
import Todos from './todos';

describe('Todos tests', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<Todos />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  test('snapshot test', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('render todos', () => {
    expect(wrapper.find('.todos-item')).toHaveLength(3);
  });
  test('delete button item', () => {
    expect(wrapper.find('.todos-item').first().childAt(1).text()).toEqual(
      'delete'
    );
  });
});
