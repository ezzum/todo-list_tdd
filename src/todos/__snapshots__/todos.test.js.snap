// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todos tests snapshot test 1`] = `ReactWrapper {}`;
